ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -1460,7 +1460,7 @@ export class WalletDB {
     return (await this.getMultisigSecretByName(name, tx)) !== undefined
   }
 
-  async *getMultisigSecrets(tx?: IDatabaseTransaction): AsyncGenerator<MultisigSecretValue> {
+  async *getMultisigSecrets(tx?: IDatabaseTransaction): AsyncGenerator<MultisigIdentityValue> {
```

Comment:
nit - should we rename these methods to go along with the rename for the datastore?